eZ/Publish/API/Repository/Tests/SearchEngineIndexingTest.php

```diff
@@ -1154,6 +1203,46 @@ protected function createContentWithNameAndDescription($contentName, $contentDes
     }
 
     /**
+     * Create and publish a content with specified, in multiple languages, fields.
+     * $parentLocationIdList.
+     *
+     * @param string[] $names multi-language name field in the form of: <code>['lang-code' => 'name']</code>
+     * @param int $parentLocationId
+     * @param bool $alwaysAvailable
+     *
+     * @return \eZ\Publish\API\Repository\Values\Content\Content
+     */
+    protected function createMultiLanguageContent(array $names, $parentLocationId, $alwaysAvailable)
+    {
+        $repository = $this->getRepository();
+        $contentService = $repository->getContentService();
+        $locationService = $this->getRepository()->getLocationService();
```

Comment:
You have a reference to repository already, so you don't need to call again  `$this->getRepository()`